### PR TITLE
[MIG] l10n_de_country_states: Migration to 12.0

### PR DESCRIPTION
**Pull request by Tardo**
Normal migration

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please add `oca_dependencies.txt` file for adding partner-contact repo and try a bit of squashing of the commits as we saw the other day.

**Comment by Tardo** [comment]
Changes done

File: l10n_de_country_states/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+{
+    'name': 'German Country States',
+    'category': 'Localisation/Europe',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'initOS GmbH'
+              'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://odoo-community.org/',
```
Comment (tbaden) [inline l10n_de_country_states/__manifest__.py]: website should be https://github.com/OCA/l10n-germany

**Review by tbaden — CHANGES_REQUESTED** [review]
LGTM, just the website tag and some commit squashing required

**Review by pedrobaeza — APPROVED** [review]
With the mentioned changed by @tbaden

File: l10n_de_country_states/__manifest__.py (hunk 1 above)
Comment (Tardo) [reply l10n_de_country_states/__manifest__.py]: Done, thx

**Comment by Tardo** [comment]
Changes done!

**Review by tbaden — APPROVED** [review]

**Pull request merged**